contracts/token/IERC721.cairo

```diff
@@ -0,0 +1,39 @@
+%lang starknet
+
+from starkware.cairo.common.uint256 import Uint256
+
+@contract_interface
+namespace IERC721:
+    func balanceOf(owner: felt) -> (balance: Uint256):
+    end
+
+    func ownerOf(token_id: Uint256) -> (owner: felt):
+    end
+
+    func safeTransferFrom(
+            _from: felt, 
+            to: felt, 
+            token_id: Uint256, 
+            data_len: felt,
+            data: felt*
+        ):
+    end
+
+    func transferFrom(_from: felt, to: felt, token_id: Uint256):
+    end
+
+    func approve(approved: felt, token_id: Uint256):
+    end
+
+    func setApprovalForAll(operator: felt, approved: felt):
+    end
+
+    func getApproved(token_id: Uint256) -> (approved: felt):
+    end
+
+    func isApprovedForAll(owner: felt, operator: felt) -> (is_approved: felt):
+    end
+
+    func supportsInterface(interface_id: felt) -> (success: felt):
```

Comment:
why was it necessary to add this?

Reply:
The short answer is: it's not. Having both `ERC165_base` and `ERC165` resolves this. I'll have the fix in #122